src/webgl/shader.js

```diff
@@ -85,7 +85,6 @@ export class Shader extends Resource {
       const {shaderName, errors, warnings} =
         parseGLSLCompilerError(infoLog, this.source, this.shaderType);
       log.error(0, `GLSL compilation errors in ${shaderName}\n${errors}`);
-      log.warn(0, `GLSL compilation warnings in ${shaderName}\n${warnings}`);
```

Comment:
This is not a fix, but rather a removal of the v4 feature of presenting warnings and errors with different "colors" in the console.

I believe that the real issue is that `parseGLSLCompilerError` includes too much text into one of the `errors` and `warnings` strings. If we could fix that problem there, then we could still show errors and warnings here with different "colors" as intended.

If you want to try to fix the error parser, the most easy way for me to reproduce this is when compiling a shader using the `fp64` module (just introduce any error to the main shader), fp64 module generates 
a number of warnings.

If you decide not to fix the parse, just comment this line out (instead of deleting it) and leave a comment that it should be restored when the parser is fixed.